cc/BUILD.bazel

```diff
@@ -1,25 +1,57 @@
-load("@modular_cc_toolchain//cc:features.bzl", "cc_feature", "action_mux")
+load("@modular_cc_toolchain//cc:features.bzl", "action_flag_mux", "cc_feature")
 load("//cc:action_names.bzl", "ACTION_NAME_GROUPS")
-load("//cc:actions.bzl", "cc_action_config")
+load("//cc:actions.bzl", "action_tool_mux", "cc_action_config")
+load("//cc:cc_toolchain_config.bzl", "cc_toolchain_config")
 load("@rules_cc//cc:action_names.bzl", "ACTION_NAMES")
 
 cc_feature(
     name = "garbage_collect_sections",
-    action_flags = action_mux({
+    enabled = True,
+    action_flags = action_flag_mux({
         # Put each function and global var in their own linker section.
         ACTION_NAME_GROUPS.all_cc_compile_actions: ["-ffunction-sections", "-fdata-sections"],
         # Remove unused functions/symbols from linked binary.
         ACTION_NAME_GROUPS.all_cc_link_actions: ["-Wl,--gc-sections"],
     }),
     doc = "Place each function in it's own section so that the linker can discard unused functions",
-    deps = [":default_clang"],
+)
+
+cc_feature(
+    name = "no_std_includes",
+    enabled = True,
+    action_flags = action_flag_mux({
+        ACTION_NAME_GROUPS.all_cc_compile_actions: [
+            "-nostdinc",
+            "-nostdinc++",
+            "-isystem/usr/include/c++/9",
```

Comment:
Why are these include paths listed both here (in this default-enabled feature) and in `cxx_builtin_include_directories`?

I guess it's just not clear to me what this feature achieves: you first exclude built-in include paths, but then add them right back in?

Reply:
It's mostly a convenience thing to make sure I have consistent includes. Sometimes clang/gcc will change/add include directories depending on other flags. This is usually true for sanitizers. But it gets rather annoying having the compiler "autodetect" different include paths and having them not matching in bazel, not to mention the error messages when this happens are rather cryptic. You end up with a lot of blackbox inconsistencies between environments. But you are probably right, it's probably not 100% necessary here and adds unnecessary indirection.

It's also an intermediate precursor to having fully hermetic libraries as well. But we aren't quite there yet with the modular toolchains.